kotest-assertions/kotest-assertions-core/src/commonMain/kotlin/io/kotest/matchers/string/empty.kt

```diff
@@ -0,0 +1,44 @@
+package io.kotest.matchers.string
+
+import io.kotest.assertions.show.show
+import io.kotest.matchers.MatcherResult
+import io.kotest.matchers.neverNullMatcher
+import io.kotest.matchers.should
+import io.kotest.matchers.shouldNot
+
+fun <A : CharSequence> A?.shouldBeEmpty(): A? {
+   this.toString() should beEmpty()
```

Comment:
hmm `null.toString()` will yield the string "null"

Reply:
Good spot.